apps/landingPage/pages/index.js

```diff
@@ -10,6 +10,10 @@ const GlobalStyle = createGlobalStyle`
     margin: 0;
     -webkit-font-smoothing: antialiased;
     font-family: 'Circular Pro', 'Roboto', sans-serif;
+
```

Comment:
I think `scroll-snap-type: y proximity;` can go here directly, as it is already `html, body {...}`

Reply:
done